packages/core/src/utils/nuxt/_proxyUtils.ts

```diff
@@ -26,16 +26,26 @@ export const createProxiedApi = ({ givenApi, client, tag }: CreateProxiedApiPara
 export const getCookies = (context: NuxtContext) => context?.req?.headers?.cookie ?? '';
 
 export const getIntegrationConfig = (context: NuxtContext, configuration: any) => {
-  const baseURL = process.server ? context?.$config?.middlewareUrl : window.location.origin;
   const cookie = getCookies(context);
 
   if (process.server && context?.$config?.middlewareUrl) {
     Logger.info('Applied middlewareUrl as ', context.$config.middlewareUrl);
   }
 
+  const { middlewareUrl, ssrMiddlewareUrl } = context.$config;
+
+  if (!middlewareUrl) {
+    throw new Error('Middleware URL is required. Provide the MiddlewareURL in your integration\'s configuration.');
```

Comment:
```suggestion
    throw new Error('`middlewareUrl` is required. Provide the `middlewareUrl` in your integration\'s configuration.');
```
Maybe like that. WDYT?

Reply:
Ok, changed.